rdt-open-reader/src/main/java/io/ona/rdt/application/RDTApplication.java

```diff
@@ -68,6 +70,11 @@ public void onCreate() {
         if (sharedPreferences.getPreference(IS_IMG_SYNC_ENABLED).isEmpty()) {
             sharedPreferences.savePreference(IS_IMG_SYNC_ENABLED, String.valueOf(true));
         }
+
+        // set locale initially
+        if (sharedPreferences.getPreference(Constants.Locale.LOCALE).isEmpty()) {
+            sharedPreferences.savePreference(Constants.Locale.LOCALE, BuildConfig.LOCALE);
+        }
```

Comment:
Why not check this through `LangUtils`?

Reply:
LangUtils return `en` by default that why I didn't use that

Reply:
Ok, use `AllConstants.LANGUAGE_PREFERENCE_KEY` then to make it uniform with the core library key for that preference.

Also, use `StringUtils.isEmpty(sharedPreferences.getPreference(AllConstants.LANGUAGE_PREFERENCE_KEY`))` instead, which is a safer check in case of NPE.